src/main/java/com/google/api/codegen/transformer/StaticLangResourceObjectTransformer.java

```diff
@@ -37,6 +37,7 @@ public RequestObjectParamView generateRequestObjectParam(
     for (FieldModel f : requiredFields) {
       if (f.getSimpleName().equals(field.getSimpleName())) {
         isRequired = true;
+        break;
```

Comment:
ooh ooh this is a good place for a streaming lambda expression (completely, utterly optional)

Reply:
I will definitely get around to using Java 8 features!

For this particular for-loop, it's iterating over an Iterable<T>, which doesn't have the .stream() method.